### PR TITLE
Auto billable checkbox

### PR DESCRIPTION
**Pull request by abinashpa**
## Preview
https://www.loom.com/share/6daacd5e59ad45ca9ea72b02ff24eb8a

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by apoorv1316** [comment]
@abinashpa CI checks are failing.

File: app/javascript/src/helpers/hhmm-parser.ts [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
   const numberDuration = Number(duration);
   if (duration.includes(":")) {
-    const [h, m] = duration.split(":");
+    let [h, m] = duration.split(":");
+    if (h.length > 2) {
+      h = h.slice(0, 2);
+    }
+    if (m.length > 2) {
+      m = m.slice(0, 2);
+    }
```
[the remark this answers is not in the document]
Comment (abinashpa) [reply app/javascript/src/helpers/hhmm-parser.ts:10]: [Notion](https://www.notion.so/saeloun/6afc9500b2cc42af86be55f37894fb09?v=f1810139870444259ef38b4bee169b34&p=e004d390dfdd4868881df39fa50a2cf5)

**Comment by apoorv1316** [comment]
@abinashpa Please take pull from develop and re-run jobs again.

**Review by apoorv1316 — APPROVED** [review]
@abinashpa Can we disable the checkbox when the project is unbilled?

**Comment by abinashpa** [comment]
> @abinashpa Can we disable the checkbox when the project is unbilled?

It's disabled by default.

**Review by supriya3105 — APPROVED** [review]

**Review by onkar-saeloun — APPROVED** [review]

**Pull request merged**